content/contributing-terms.en.md

```diff
@@ -3,539 +3,29 @@ title: "Contributing terms"
 
-Before adding new terms, open the [`declarations`](https://github.com/OpenTermsArchive/contrib-declarations/tree/main/declarations) folder and check if the service you want to track terms for is already declared. If a JSON file with the name of the service is already present, you can jump straight to [declaring terms](#declaring-terms). Otherwise, keep reading!
+## Prerequisites
 
-### Declaring a new service
+In order to add declarations:
 
-Before declaring a service, you will need to choose the service name and service ID. The service ID will be the name of the JSON file in which the service will be declared. It is a normalised version of the service name.
+1. You need to have [Node.js](https://nodejs.org/en/) installed on your machine. If you don't have it, you can download it from the official website [here](https://nodejs.org/en/download/).
+2. You need to have git installed on your machine. If you don't have it, you can download it from the official website [here](https://git-scm.com/downloads).
 
-### Service name
+## Adding a declaration
 
-The service name is exposed to end users. It should reflect as closely as possible the official service name, as referenced in the terms or “about” pages, so that it can be recognised easily and unambiguously.
+To add a declaration, you need to follow these steps:
 
-- The service name should be the one used by the service itself, no matter the alphabet, UTF symbols, spaces, and casing.
-  - _Example: `eBay`_.
-  - _Example: `hi5`_.
-  - _Example: `LINE`_.
-  - _Example: `App Store`_.
-  - _Example: `туту.ру` (Cyrillic)_.
-  - _Example: `抖音短视频` (Simplified Chinese characters)_.
-- Domain name extensions (TLDs) are allowed when they are part of the official service name.
-  - _Example: `Booking.com`_.
-  - _Example: `historielærer.dk`_.
-- Service names can be prefixed by the provider name, separated by a space, when it would otherwise be too generic or ambiguous.
-  - _Example: `Ads` (by Facebook) → `Facebook Ads`_.
-  - _Example: `Analytics` (by Google) → `Google Analytics`_.
-  - _Example: `Firebase` (by Google) → `Firebase`_.
-  - _Example: `App Store` (by Apple) → `App Store`_.
+1. Clone the [contrib-declaration](https://github.com/OpenTermsArchive/contrib-declarations) repository to your local machine.
+2. Create a branch that describes your contribution e.g. `add-Open-Terms-Archive-ToS` or `add-firefox-privacy-policy`
+3. Run `npm install`. This will install all the dependencies including the Open Terms Archive engine which will allow you to test and validate your declaration.
+4. Create a JSON file with, as filename, the service ID of the service you are adding the declaration for. This JSON file should be in the `declarations` folder of the repository. To learn more about selecting the right service ID, please read the [declaring a new service]({{< relref "reference/declaration#declaring-a-new-service" >}}) section.
```

Comment:
Add links to the Declaration reference page for “Service ID”.

Reply:
Added inline link to that